db-server/index.js

```diff
@@ -113,7 +113,6 @@ function createServer(db) {
     })
   )
 
-  api.get('/sessionToken/:id', withIdAndBody(db.sessionToken))
```

Comment:
From an API standpoint, I wonder if it's worth keeping this endpoint and just changing it to call `db.sessionWithDevice`; it feels a bit odd to have a resource with `put` and `del` but no `get`.

Reply:
(IOW, the plain "get session token" method exposed by this API, should just always include the device and verification info)

Reply:
That seems like a reasonable change to me. We might also be able to replace corresponding [auth-server](https://github.com/mozilla/fxa-auth-server/blob/c805f9c334b99c1162f3389fc5c98d0b5ad602e5/lib/db.js#L535) call to make it less clunky.

Reply:
Agree with this. Would also change the name of the db method to `.sessionToken`, the same way we did in the auth server. (wouldn't change the stored procedure name though, just for the sake of sane versioning if we ever bring back non-device procedure for some reason)